crates/bevy_sprite/src/image.rs

```diff
@@ -0,0 +1,31 @@
+use crate::TextureAtlas;
+use bevy_asset::Handle;
+use bevy_ecs::component::Component;
+use bevy_reflect::Reflect;
+use bevy_render::texture::{Image, DEFAULT_IMAGE_HANDLE};
+
+/// The sprite texture
+#[derive(Component, Clone, Debug, Reflect)]
+pub enum SpriteImage {
```

Comment:
Perhaps just `Sprite`? I'm curious about others opinion here.

Reply:
Well `Sprite` is already a component